diff --git a/annotations.py b/annotations.py
index a9d7d63..28cd008 100644
--- a/annotations.py
+++ b/annotations.py
@@ -313,8 +313,6 @@ def _get_var_ref_and_alt(var):
     """
     if isinstance(var, cyvcf2.Variant):
         return var.REF, var.ALT
-    if isinstance(var, pysam.VariantRecord):
-        return var.ref, var.alts
     if isinstance(var, basestring):
         # Assume var is a line from a VCF.
         ref, alt = var.split('\t', 6)[3:5]
@@ -419,7 +417,7 @@ def annotations_in_vcf(var, anno, parser_type=None, naming="ucsc", region_only=F
         # Filter hits to those that match ref and alt.
         matched_hits = []
         for h in hits:
-            if isinstance(h, (cyvcf2.Variant, pysam.VariantRecord)):
+            if isinstance(h, cyvcf2.Variant):
                 start = h.start
             elif isinstance(h, basestring):
                 start = int(h.split('\t', 2)[1]) - 1
